torchvision/transforms/autoaugment.py

```diff
@@ -3,7 +3,7 @@
 
 from enum import Enum
 from torch import Tensor
-from typing import List, Tuple, Optional
+from typing import List, Tuple, Optional, Dict, Any
```

Comment:
I think the `Any` is not being used anymore?

Reply:
Francisco is right:
```suggestion
from typing import List, Tuple, Optional, Dict
```